internal/handlers/publishers.go

```diff
@@ -65,3 +66,15 @@ func (h *PublisherHandler) CreatePublisherHandler(w http.ResponseWriter, r *http
 	responseMessage := fmt.Sprintf("Editora criada com sucesso: %s", name)
 	w.Write([]byte(responseMessage))
 }
+
+func (h *PublisherHandler) NewPublisherForm(w http.ResponseWriter, r *http.Request) {
+	page, err := renderer.HTML.Render("publishers/new.html", nil)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("Ocorreu um erro ao renderizar a página"))
```

Comment:
```suggestion
		http.Error(w, "Ocorreu um erro ao renderizar a página", http.StatusInternalServerError)
```

Dá para usar o [`http.Error()`](https://pkg.go.dev/net/http#Error) para simplificar um pouco o tratamento de erro.

Reply:
feito, [5c3de84be1b12d8e673b1186a4a3e2ab381e61f1](https://github.com/codando-junto/lucienne/pull/106/commits/5c3de84be1b12d8e673b1186a4a3e2ab381e61f1)